x-pack/test/security_solution_cypress/cypress/e2e/detection_response/detection_engine/exceptions/rule_details_flow/add_edit_exception_data_view.cy.ts

```diff
@@ -48,7 +48,7 @@ import { waitForAlertsToPopulate } from '../../../../../tasks/create_new_rule';
 const DATAVIEW = 'auditbeat-exceptions-*';
 
 // Failing: See https://github.com/elastic/kibana/issues/207913
```

Comment:
Will this comment be removed automatically? or should we do it manually in this PR?

Reply:
I did miss this initially, but it was removed in a subsequent commit (you're just looking at a single commit, here).